Automattic/wp-calypso
# State: Introduce action to update WordPress core

This PR introduces the Redux action & reducer to update the WordPress core of a Jetpack site. It will be used to replace the `JetpackSite.prototype.updateWordPress` method and the attributes that it manages (`updating`) as we migrate away from `sites-list`.

To test:
* Checkout this branch
* Verify all tests pass:

```
npm run test-client client/state/sites/updates
```
